### PR TITLE
angular2 authorization

### PR DESCRIPTION
**Pull request by alemata**
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [ ] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [ ] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 

If you are an Auth0 employee, after you submit this PR create and test a [Review App](https://github.com/auth0/docs#review-apps) to test your changes.
### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by alemata** [comment]
@flaviogranero could you review please?

File: articles/client-platforms/angular2/08-authorization.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+...
+public isAdmin() {
+  return this.userProfile && this.userProfile.app_metadata
+    && this.userProfile.app_metadata.roles
+    && this.userProfile.app_metadata.roles.indexOf('admin') > -1;
+}
+...
+```
+
+That's it. Only ff you login with a mail that contains `@example` you will be able to access `/admin` route.
```
Comment (flaviogranero) [inline articles/client-platforms/angular2/08-authorization.md]: -> Only if

**Comment by alemata** [comment]
@flaviogranero, @christiannwamba I updated using a include for the introduction, so we can all use the same.

what do you think? Feel free to improve anything

**Comment by flaviogranero** [comment]
sounds good! 👍  @ntotten

**Pull request merged**